uaclient/tests/test_cli_disable.py

```diff
@@ -100,7 +100,7 @@ def factory_side_effect(name, ent_dict=ent_dict):
         m_entitlement_factory.side_effect = factory_side_effect
 
         m_cfg = mock.Mock()
-        m_cfg.check_lock_info.return_value = (-1, "")
+        m_cfg.check_lock_info.return_value = (-2, "")
```

Comment:
Curious: why is this change necessary? It's hard to tell here

Reply:
Good catch, I honestly think this is a mistake on my end. I will update the code

Reply:
All good thanks!